plottable.d.ts

```diff
@@ -2671,8 +2673,8 @@ declare module Plottable {
              * Constructs a RectanglePlot.
              *
              * A RectanglePlot consists of a bunch of rectangles. The user is required to
-             * project the left and right bounds of the rectangle (x1 and x2 respectively)
-             * as well as the bottom and top bounds (y1 and y2 respectively)
+             * project the left and right bounds of the rectangle (x and x1 respectively)
+             * as well as the bottom and top bounds (y and y1 respectively)
```

Comment:
Should probably mention about the behavior if only x is set etc. (You can also leave it until the API calls onto `x()`, `y()`).  That said, I wouldn't worry too much about the documentation as there's a ton of documentation that we'll have to write in the next week or so.